example/app.jsx

```diff
@@ -53,10 +55,27 @@ function App() {
       </Fragment>
     );
   }
+  const synth1 = useMemo(
```

Comment:
Yes, it could be however that would defy the purpose of the example altogether. With this added synth example I aimed to showcase how P5Wrapper handles large objects as props and the effect that the new added deepCompare prop will have on that. I chose the tone.js synths as my large objects  as they are long enough to cause significant delays when deep diffing. That's why its necessary that the synth instances live on `App.jsx`, so that they can actually be passed down to the synth sketch through props. Otherwise the deepCompare prop wouldn't be put to action.
hope that makes sense